index.html

```diff
@@ -1,313 +1,299 @@
-            <div class="lang-arrow">
-              <h3 class="lang-text">Frameworks</h3>
-              <img
-                class="lang-arrow-right"
-                src="assets/images/Union.svg"
-                alt="arrow"
-              />
-            </div>
-            <hr />
+          <li class="social-icons">
+            <a href="https://www.linkedin.com/in/vikipretium/"><img src="assets/images/l.svg" alt="linkedin" /></a>
           </li>
-          <li class="lang-div">
-            <div class="lang-arrow">
-              <h3 class="lang-text">Skills</h3>
-              <img
-                class="lang-arrow-right"
-                src="assets/images/Union.svg"
-                alt="arrow"
-              />
-            </div>
-            <hr />
+          <li class="social-icons">
+            <a href="https://angel.co/u/vikipretium"><img src="assets/images/a.svg" alt="Angel" /></a>
+          </li>
+          <li class="social-icons">
+            <a href="https://twitter.com/vikipretium"><img src="assets/images/t.svg" alt="twitter" /></a>
+          </li>
+          <li class="social-icons">
+            <a href="https://medium.com/@vighneshpasupathy"><img src="assets/images/m.svg" alt="medium" /></a>
           </li>
         </ul>
-      </section>
-      <!-- Languages skills frameworks ends -->
-    </main>
+      </div>
+      <button type="button" class="button">Get my resume</button>
 
-    <footer>
-      <h2>Contact me</h2>
-      <p>
+      <!-- Languages skills frameworks starts-->
+      <ul class="list-lang">
+        <li class="lang-div">
+          <div class="lang-arrow">
+            <h3 class="lang-text">Languages</h3>
+            <img src="assets/images/Union.svg" alt="arrow" />
+          </div>
+          <div class="lang-wrapper">
+            <div class="lang">
+              <img src="assets/images/js.png" alt="javascript" />
+              <h4>Javascript</h4>
+            </div>
+            <div class="lang">
+              <img src="assets/images/html.png" alt="html" />
+              <h4>HTML</h4>
+            </div>
+            <div class="lang">
+              <img src="assets/images/css.png" alt="css" />
+              <h4>CSS</h4>
+            </div>
+          </div>
+        </li>
+
+        <li class="lang-div">
+          <div class="lang-arrow">
+            <h3 class="lang-text">Frameworks</h3>
+            <img class="lang-arrow-right" src="assets/images/Union.svg" alt="arrow" />
+          </div>
+          <hr />
+        </li>
+        <li class="lang-div">
+          <div class="lang-arrow">
+            <h3 class="lang-text">Skills</h3>
+            <img class="lang-arrow-right" src="assets/images/Union.svg" alt="arrow" />
+          </div>
+          <hr />
+        </li>
+      </ul>
+    </section>
+    <!-- Languages skills frameworks ends -->
+  </main>
+
+  <footer>
+    <div class="contact-form">
+      <h2 class="form-heading">Contact me</h2>
```

Comment:
- [ ] Kindly make sure all backgrounds designs are implemented to match the Figma design 👍☺️
Your Design | Figma Design
:---:|:---:
![](https://user-images.githubusercontent.com/44624138/149272230-c3dc4d35-f48a-4820-8a4e-f8e728787c1e.png) | ![](https://user-images.githubusercontent.com/44624138/149272206-d1067ad0-936a-445b-9edf-350bde8cf073.png)

Reply:
Hello @rotimiazeez,

Reply:
My mistake @rotimiazeez, 
![image](https://user-images.githubusercontent.com/20333314/149276088-0364898f-6aca-481e-a456-383bc5660daa.png)
I just entered the file format two times 😅 .